dist/BrazeKit.common.js

```diff
@@ -9172,7 +9172,7 @@ window.braze = require$$0;
 var name = 'Appboy',
     suffix = 'v4',
     moduleId = 28,
-    version = '4.1.0',
+    version = '4.1.1',
```

Comment:
Please undo this change. Our process is to update the version number and distribution as a separate commit when we release.

Reply:
This change seems to already be [in the master-v4 branch](https://github.com/mparticle-integrations/mparticle-javascript-integration-braze/blob/b043d4637a7e0b6d82bf550d63936a77d6b2e0c2/dist/BrazeKit.common.js#L9175). I don't think it was a change that I made. Not sure I know how to roll it back in this case.

Reply:
OK, you may need to rebase your branch against `master-v4`. I'll see if I can help you out with that.